test/Tizen.NUI.Samples/Tizen.NUI.Samples/SampleMain.cs

```diff
@@ -11,16 +11,19 @@ public class CommonResource
         public static string GetDaliResourcePath()
         {
             return @"/opt/usr/globalapps/org.tizen.example.Tizen.NUI.Samples/res/images/Dali/";
+            //return @"/home/owner/apps_rw/org.tizen.example.Tizen.NUI.Samples/res/images/Dali/";
```

Comment:
After replacing with "Tizen.Applications.Application.Current.DirectoryInfo.Resource", app will crash
System.NullReferenceException: Object reference not set to an instance of an object. Because the app has not initialized.

Reply:
I think it can be caused by adding style json file when initializing application.
Actually, you can do that after application created using StyleManager.

Could you SetTheme after application created(maybe in OnCreate method)

Reply:
Fixed